packages/app/src/app/components/CreateSandbox/Import/AccountSelect.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from 'styled-components';
+import VisuallyHidden from '@reach/visually-hidden';
+import {
+  useMenuState,
+  Menu,
+  MenuItem,
+  MenuDisclosure,
+  MenuSeparator,
+} from 'reakit/Menu';
+
+import { Stack, Text, Icon, Element } from '@codesandbox/components';
+import { useActions } from 'app/overmind';
+
+// In the new codebase this should be a Radix dropdown menu
+// https://www.radix-ui.com/docs/primitives/components/dropdown-menu
+// also, more composable if it were a design system component
+export const AccountSelect = ({ options, value, onChange }) => {
+  const { signInGithubClicked } = useActions();
+  const menu = useMenuState();
+
+  return (
+    <>
+      <StyledMenuButton {...menu}>
+        <Stack gap={2} align="center">
+          <Icon name="github" />
+          <Stack gap={2} align="center">
+            <VisuallyHidden>Select GitHub account.</VisuallyHidden>
+            <Text size={13}>
+              {value} <VisuallyHidden>is currently selected</VisuallyHidden>
+            </Text>
+            <Icon name="chevronDown" size={8} />
+          </Stack>
+        </Stack>
+      </StyledMenuButton>
+      <StyledMenu {...menu} aria-label="GitHub account selection">
```

Comment:
Maybe the `aria-label` isn't needed? The menu is controlled by the button and I imagine that would be enough.

Reply:
The button is indeed enough for the button, but it's common to label `role="menu"` elements too! We could indeed use the button for this with `labelledby`, I think Radix does that, but I'm not sure.